core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -155,6 +160,45 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
     adminClient.listTopics().names().get().contains(testTopicName)
   }
 
+  @Test
+  def testCreateWithDefaults(): Unit = {
+    createAndWaitTopic(new TopicCommandOptions(
+      Array("--topic", testTopicName)))
+
+    adminClient.listTopics().names().get().contains(testTopicName)
```

Comment:
Can we add default assertions here. Maybe we can just use `describeTopic` like the other tests?

Reply:
Oops i missed this comment! Addressing that now.